test/e2e/mdc_test.go

```diff
@@ -30,6 +36,63 @@ func MdcTest(t *testing.T) {
 	ctx := framework.NewTestCtx(t)
 	defer ctx.Cleanup()
 
+	f := framework.Global
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		t.Fatalf("failed to get namespace: %v", err)
+	}
+
+	mdcAppName := "mdc"
+	mdcTestCR := &mdcv1alpha1.MobileDeveloperConsole{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "MobileDeveloperConsole",
+			APIVersion: "mdc.aerogear.org/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      mdcAppName,
+			Namespace: namespace,
+		},
+		Spec: operator.MobileDeveloperConsoleSpec{
+			OAuthClientId:     "mobile-developer-console",
+			OAuthClientSecret: "dummy-client-secret",
+		},
+	}
+
+	if err := initializeMDCResources(t, f, ctx, namespace); err != nil {
+		t.Fatal(err)
+	}
+
+	// Create MDC CR
+	if err := createMDCCustomResource(t, f, ctx, mdcTestCR); err != nil {
```

Comment:
I think the fact that you've got such a nice clear descriptive function name here, the comment is maybe not necessary (but doesn't cause any harm either). :)